### ISSUE TITLE
Time based blocking

### ISSUE THREAD

**Issue by Ashwin-droid** [AdguardTeam/AdGuardHome#3067]
Have a question or an idea? Please search it [on our forum](https://github.com/AdguardTeam/AdGuardHome/discussions) to make sure it was not yet asked. If you cannot find what you had in mind, please [submit it here](https://github.com/AdguardTeam/AdGuardHome/discussions/new).

### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed

### Problem Description

Usually at night after you sleep there is mostly tracking and syncing going on or blocking more services on weekdays for specific clients for focus mode and unblocking them at a break time etc

### Proposed Solution

A page in management to set time based blocking and unblocking

**Comment by ainar-g** [comment]
Duplicate of #1203.
